Contributor.md

```diff
@@ -1,3 +1,9 @@
+- Name: Augusto Moraes
```

Comment:
Please move your info to the bottom of the list.

Reply:
done @jinwoov